CHANGELOG.rst

```diff
@@ -13,13 +13,16 @@ User-Facing Changes
 
 **🌟 New Features**
 
-* 📑 **UI**: Add a new dashboard for logged-in users, showing running sessions, last
-  accessed projects, and own datasets
-  (`#2332 <https://github.com/SwissDataScienceCenter/renku-ui/pull/2332>`_).
 * ⚙️ **Renku CLI**: Connecting to Renkulab sessions via SSH
   (`#3318 <https://github.com/SwissDataScienceCenter/renku-python/pull/3318>`_).
 * 📃 **Renku CLI**: Support for Azure Blob Storage backed datasets.
   (`#3257 <https://github.com/SwissDataScienceCenter/renku-python/pull/3257>`_).
+* 🧑‍💻 **UI**: Add a new dashboard for logged-in users, showing running sessions, last
+  accessed projects, and own datasets
+  (`#2332 <https://github.com/SwissDataScienceCenter/renku-ui/pull/2332>`_).
+* 📑 **UI**: Show instructions on how to connect to RenkuLab sessions via SSH from the
```

Comment:
I already asked about this but I guess I did not really understand the answer: where is this option toggled? In other words we don't need a parameter in the values to tell UI to whether to show or not these instructions?

Reply:
The UI gets it from notebook service which has it set here: https://github.com/SwissDataScienceCenter/renku-notebooks/blob/master/helm-chart/renku-notebooks/values.yaml#L340

Reply:
great, thanks! The release of notebooks containing that still needs to be added here?

Reply:
Nope, that's not necessary. The UI invokes the `GET /version` API on renku-notebooks to check whether SSH is actually supported or not

Reply:
@aledegano You may be referring to the notebooks release PR https://github.com/SwissDataScienceCenter/renku/pull/2967, which is also on its way into this `feat/ssh-into-sessions` branch.

Reply:
(sorry haven't refreshed the page before answering)
All will be merged in the feature PR that this PR is also referencing